runtime/android/core_internal/src/org/xwalk/core/internal/XWalkUIClientInternal.java

```diff
@@ -522,4 +531,100 @@ public void onCancel(DialogInterface dialog) {
         mDialog.show();
         return false;
     }
+
+    void setContentsClient(XWalkContentsClient client) {
+        mContentsClient = client;
+    }
+
+    private Activity addContentView(View view, CustomViewCallbackInternal callback) {
+        Activity activity = mXWalkView.getActivity();
+        if (mCustomXWalkView != null || activity == null) {
+            if (callback != null) callback.onCustomViewHidden();
```

Comment:
The callback will be Invoked when the host application dismisses the custom view explained in https://developer.android.com/reference/android/webkit/WebChromeClient.CustomViewCallback.html, customer need to exit full screen mode.
The simple code in java/src/org/chromium/android_webview/AwContentViewClient.java
```
WebChromeClient.CustomViewCallback cb = new WebChromeClient.CustomViewCallback() {
            @Override
            public void onCustomViewHidden() {
                if (mCustomView != null) {
                    mAwContents.requestExitFullscreen();
                }
            }
        };
```
But above logic isn't align with it's usage?